src/services/download/sentry.rs

```diff
@@ -144,9 +148,11 @@ async fn fetch_sentry_json(query: &SearchQuery) -> Result<Vec<SearchResult>, Sen
 ///
 /// If there are cached search results this skips the actual search.
 async fn cached_sentry_search(query: SearchQuery) -> Result<Vec<SearchResult>, DownloadError> {
-    if let Some((created, entries)) = SENTRY_SEARCH_RESULTS.lock().get(&query) {
-        if created.elapsed() < Duration::from_secs(3600) {
-            return Ok(entries.clone());
+    if let Some(cache_duration) = query.index_cache_ttl {
```

Comment:
The `index_cache_ttl` should just be `Duration` as `Duration::default()` is 0. Particularly, this gets rid of a branch here, and the comparison below still works.

Reply:
I did this, the entire config reading is now in this function, so the amount of conditionals _in tihs function_ did not decrease as it gained the conditionals that were previously in the downloads actor

Reply:
yeah no dice, even our own Worker requires static even though it has new-style futures